application/jaxrs-mongodb-quarkus/src/test/java/webapp/tier/resource/MongodbResourceTest.java

```diff
@@ -1,45 +1,64 @@
 package webapp.tier.resource;
 
-import static io.restassured.RestAssured.*;
-import static org.hamcrest.CoreMatchers.*;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.containsString;
+import static org.mockito.Mockito.when;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.List;
 
 import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
 
 import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectMock;
+import webapp.tier.bean.MsgBean;
+import webapp.tier.service.MongodbService;
+import webapp.tier.util.CreateId;
 
 @QuarkusTest
 class MongodbResourceTest {
 
+	@InjectMock
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5923183)